client/src/Profile.js

```diff
@@ -0,0 +1,118 @@
+    document.getElementById("username").disabled = !state.isDisable
+    document.getElementById("password").disabled = !state.isDisable
+
+    if (state.isDisable === true) {
+        // document.getElementById("editLabel").textContent = "Done"
+        document.getElementById("doneButton").style.display = "inline"
+        document.getElementById("editButton").style.display = "none"
+        state.isDisable = false
+    } else {
+
+        //submit changes
+        //update database here
+
+        //document.getElementById("editLabel").textContent = "Edit"
+        document.getElementById("editButton").style.display = "inline"
+        document.getElementById("doneButton").style.display = "none"
+        state.isDisable = true
+    }
+}
+
+export default function Profile() {
+    const classes = styles();
+    return (
+        <Grid container className={classes.root}>
+            <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
+                <div className={classes.paper}>
+                    <form className={classes.form} noValidate id="submitForm">
+                        <label id="welcome">Personal Information</label>
+                        <Button onClick= {handleEdit} id="editButton">[Edit]</Button>
+                        <Button onClick= {handleEdit} id="doneButton">[Done]</Button>
+                        {/* <IconButton onClick= {handleEdit} id="editIcon">
+                            <CreateIcon></CreateIcon>
+                        </IconButton>
+
+                        <IconButton onClick= {handleEdit} id="checkIcon">
+                            <CheckIcon></CheckIcon>
+                        </IconButton> */}
+
+                        {/* <label id="editLabel">Edit</label> */}
+
+                        <TextField
+                            id="firstname"
+                            label="First name"
+                            margin="normal"
+                            fullWidth
+                            disabled = 'true'
+                        />
+
+                        <TextField
+                            id="lastname"
+                            label="Last name"
+                            margin="normal"
+                            fullWidth
+                            disabled = 'true'
+                        />
+
+                        <TextField
+                            id="username"
+                            label="Username"
+                            margin="normal"
+                            fullWidth
+                            disabled = 'true'
+                        />
+
+                        <TextField
+                            id="password"
+                            label="Password"
+                            margin="normal"
+                            fullWidth
+                            type='password'
+                            disabled = 'true'
+                        />
```

Comment:
Can we move the password to another section, kind of like this:
![image](https://user-images.githubusercontent.com/36905285/68989077-a502e100-07f6-11ea-9e95-0b3e90e3c8b0.png)

Reply:
okay sounds good